x-pack/plugins/security_solution/cypress/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "target/**/*"
   ],
   "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom"],
```

Comment:
This is interesting changes, does this give us new abilities such as being able to use `??` where before we couldn't?

Reply:
From chat this does seem to be the case 👍